samples/deid.js

```diff
@@ -1,609 +0,0 @@
-// Copyright 2017 Google LLC
```

Comment:
So the moment you delete a sample here, the ones on cloudsite could be orphaned.  You may need to go update all fo the paths in http://devrel/samples as soon as this is landed, otherwise it will break a bunch of stuff on cloud.google.com.  I'd reach out to @fhinkel to make sure we do this right :)

Reply:
Ah, I know how to do this. I forgot about this part. I'll make sure to remember to do it soon after!